atheaderservice/Dockerfile

```diff
@@ -0,0 +1,77 @@
+ARG LSSTTS_SAL_VERSION
+ARG LSSTTS_XML_VERSION
+ARG LSSTTS_SALOBJ_VERSION
+FROM lsstdm/salobj:${LSSTTS_SAL_VERSION}-${LSSTTS_XML_VERSION}-${LSSTTS_SALOBJ_VERSION}
+
+LABEL name="LSST AuxTel Header Service Development Environment" \
+    vendor="LSST" \
+    license="GPLv3"
+
+# These ARGS need to be re-declared
+ARG LSSTTS_SAL_VERSION
+ARG LSSTTS_XML_VERSION
+ARG LSSTTS_RPM_VERSION=$LSSTTS_SAL_VERSION-$LSSTTS_XML_VERSION.el7
+
+# Versions
+ARG HEADERSERVICE_VERSION=1.0.1
```

Comment:
By embedding versions here, even if they can be overridden, you make it more error-prone to reuse this.  I would suggest leaving out the default for this primary version.

Reply:
I've change it so that now both `HEADERSERVICE_VERSION` and `SALPYTOOLS_VERSION` are environment variables.